src/components/Details/Details.js

```diff
@@ -0,0 +1,135 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import {
+  ScrollView,
+  Text,
+  ImageBackground,
+  View,
+  Image,
+  ActivityIndicator,
+} from 'react-native';
+import { useTheme } from '@react-navigation/native';
+import { PropTypes } from 'prop-types';
+
+import styles from './styles';
+import Center from 'components/Center';
+import { fetchGenres } from 'controllers/MoviesClient';
+import MovieInfo from 'components/MovieInfo/MovieInfo';
+import textStyles from 'helpers/TextStyles';
+
+const Details = ({ route }) => {
+  const [genres, setGenres] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
This shouldn't be needed, check https://github.com/moove-it/react-native-template/blob/master/src/selectors/StatusSelectors.js for how to use status selectors